common/txmgr/broadcaster.go

```diff
@@ -558,21 +558,32 @@ func (eb *Broadcaster[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) hand
 		eb.sequenceTracker.GenerateNextSequence(etx.FromAddress, *etx.Sequence)
 		return err, true
 	case client.Underpriced:
-		return eb.tryAgainBumpingGas(ctx, lgr, err, etx, attempt, initialBroadcastAt, retryCount+1)
```

Comment:
I think we'd still need to keep the call to `tryAgainBumpingGas` since it has other stuff in there like logging and creating the bump attempt. We would just want to pull the `handleInProgressTx` call out of `saveTryAgainAttempt`.

Reply:
Oh sorry I think I confused myself with the method renaming

Reply:
mb let me add some comments